passes/techmap/abc.cc

```diff
@@ -1674,6 +1674,8 @@ struct AbcPass : public Pass {
 			}
 			if (arg == "-dff") {
 				dff_mode = true;
+				if (delay_target.empty())
+					delay_target = "-D 1";
```

Comment:
Why not just call `abc` with `-D 1` in `synth_xilinx`? Forcing this default value seems to remove flexibility. And `1 ps` really doesn't seem like a good default value, especially for things like `upsize` and `dnsize`.

Reply:
The intention was to make it work across the board (for all `synth_* -retime`) and also for `abc -dff` when a timing target was not specified.
I assume ASIC users would normally be expected to provide a delay target, no?